content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix  
+1. Clone the [documentation repository](https://github.com/sodafoundation/documentation.git)  
+2. Navigate to the folder structure content/support-matrix/delfin  
+3. Create a new folder with `vendor` name. For example `dell-emc`.  
+4. Create a new file `_index.md`  
+5. Add the frontmatter as follows to add new Vendor. Change the content as required.  
+
+```
+---
+title: Dell EMC
+description: "Support matrix data for Dell EMC Models can be found here"
+weight: 5
+disableToc: false
+---
+
+Support matrix data for Dell EMC Models can be found here
+
+```
+{{% notice note %}}
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+{{% notice note %}}
+Step 3, 4 and 5 are not required if there is already vendor folder name exists in the documentation repository
+{{% /notice %}}  
+6. Under vendor folder name, create a new folder with `model` name. For example `VNX`.  
```

Comment:
nit: `VNX` -> `vnx`
We use lowercase letters for folder names.

Reply:
done